src/colvarcomp.h

```diff
@@ -28,6 +28,11 @@
 #include "colvar.h"
 #include "colvar_geometricpath.h"
 
+#ifdef TORCH
+#include <torch/torch.h>
+#include <torch/script.h>
```

Comment:
Maybe @giacomofiorin would prefer PIMPL for not directly including pytorch headers. I am fine with it anyway.

Reply:
I think this is fine, too. We have a similar example with Lepton in `colvar.h`.

Reply:
Going back to this, I think it would actually be better to isolate the `#include <torch/*.h>` into a source file. Now that the Torch interface is always included in the CI jobs, we have a 3-fold increase in build times when the compiler cache is invalidated.

Reply:
@giacomofiorin I added a separate header file `colvarcomp_torchann.h` to include the torch dependent code. Now COLVARS_TORCH only affects `colvarscomp_torchann.h`, `colvarscomp_torchann.cpp`, and `colvar.cpp`.
The library build time reduces from 23m 44s to  16m 59s.  Are these changes OK?